backend/scripts/set_src_dest_envs.sh

```diff
@@ -3,20 +3,34 @@
 export SRC_ENV=${1:-prod}
 export DEST_ENV=${2:-dev}
 
+# require DEST_ENV=rdev when STACK is set
+if [[ ! -z "$STACK" ]]; then
+    if [[ ! "$DEST_ENV" == rdev ]]; then
+        echo "ERROR: when STACK var is set, DEST_ENV must be 'rdev'"
+        exit 1
+    fi
+fi
+
 # allow dev->dev for testing
 if [[ "$SRC_ENV" == dev && "$DEST_ENV" == dev ]]; then
     if [[ -z $NO_PROMPT ]]; then
         read -n 1 -p "ATTENTION: You are mirroring the dev env to itself. Useful for testing the mirroring operation, but otherwise useless. Proceed? (Y/n) " ANS
         [[ $ANS == 'Y' ]] || exit 1
     fi
+# allow dev->rdev for testing
+elif [[ "$SRC_ENV" == dev && "$DEST_ENV" == rdev ]]; then
+    if [[ -z $NO_PROMPT ]]; then
+        read -n 1 -p "ATTENTION: You are mirroring the dev env to rdev. Useful for testing fixes in rdev for data-related bugs found in dev, but otherwise useless. Proceed? (Y/n) " ANS
+        [[ $ANS == 'Y' ]] || exit 1
+    fi
```

Comment:
same as earlier comment, mirroring _from_ dev not useful post-CI/CD

Reply:
Agreed -- shall we remove this only once we actually get rid of dev?

Reply:
if you think we'll remember...